roboteam_ai/src/skills/SideAttacker.cpp

```diff
@@ -0,0 +1,75 @@
+//
+// Created by thijs on 17-12-18.
+//
+
+#include "SideAttacker.h"
+
+namespace rtt {
+namespace ai {
+
+SideAttacker::SideAttacker(string name, bt::Blackboard::Ptr blackboard)
+        :Skill(name, blackboard) {
+}
+
+/// Init the GoToPos skill
+void SideAttacker::onInitialize() {
+    robot = getRobotFromProperties(properties);
```

Comment:
This is not needed. the robot is already received in Initialize in Skill, which means that you don't have to define the onInitialize function here.